### PR TITLE
Cleanup sample server

### PR DESCRIPTION
**Pull request by johngmyers**
Cleanup sample server

### CODE REVIEW

File: sample-server/src/main/java/com/proofpoint/platform/sample/StoreConfig.java [hunk 1]
```diff
@@ -37,9 +38,7 @@ StoreConfig setTtlInMs(int duration)
     @Config("store.ttl")
     StoreConfig setTtl(Duration ttl)
     {
-        Preconditions.checkNotNull(ttl, "ttl must not be null");
-        Preconditions.checkArgument(ttl.toMillis() > 0, "ttl must be > 0");
-
+        checkNotNull(ttl, "ttl must not be null");
```
Comment (gwittel) [inline sample-server/src/main/java/com/proofpoint/platform/sample/StoreConfig.java]: Any particular reason this isn't just done on the getter with a ```@NotNull``` validator?

Comment (johngmyers) [inline sample-server/src/main/java/com/proofpoint/platform/sample/StoreConfig.java]: The default is not null, so there's no legit way for this to be null.

**Comment by gwittel** [comment]
LGTM

**Comment by johngmyers** [comment]
Rebased and merged

**Pull request closed without merging**